libraries/chain/include/eos/chain/config.hpp

```diff
@@ -53,6 +61,7 @@ const static ShareType DefaultElectedPay = Asset(100).amount;
 const static ShareType DefaultRunnerUpPay = Asset(75).amount;
 const static ShareType DefaultMinEosBalance = Asset(100).amount;
 const static UInt32 DefaultMaxTrxLifetime = 60*60;
+const static double ProducersAuthorityThreshold = 2.0/3.0;
```

Comment:
we should never use double's in our code, especially any logic that is run as part of consensus.